gaps.exceptions.gapsExecutionError
==================================

.. currentmodule:: gaps.exceptions

.. autoexception:: gapsExecutionError